source/_docs/terminus/examples.md

```diff
@@ -129,6 +130,53 @@ The following command will permanently delete all uncommitted SFTP changes. If y
 <figure><pre id="mass-update-apply"><code class="command bash" data-lang="bash">terminus site:list --format=list | terminus site:mass-update:apply --accept-upstream --updatedb</code></pre></figure>
 </div>
 
+## Switch Upstreams
+Every site has an upstream assigned in order to deliver [one-click updates](/docs/upstream-updates/) in the Pantheon Site Dashboard. Terminus can be used to manage this site level configuration.
+### Move an existing site to a Custom Upstream
+<p class="instruction">If your organization has a <a href="/docs/custom-upstream/">Custom Upstream</a>, you can use Terminus to switch existing sites over to the common codebase:</p>
+<div class="copy-snippet">
+<button class="btn btn-default btn-clippy" data-clipboard-target="#upstream-set">Copy</button>
+<figure><pre id="upstream-set"><code class="command bash" data-lang="bash">terminus site:upstream:set my-site "My Custom Upstream"</code></pre></figure>
```

Comment:
Should this call out it's using the site machine name and the upstream machine name?